src/main/java/cz/xtf/radanalytics/util/configuration/RadanalyticsConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package cz.xtf.radanalytics.util.configuration;
+
+import cz.xtf.XTFConfiguration;
+
+public class RadanalyticsConfiguration extends XTFConfiguration {
+	private static final String IMAGE_PY_SPARK = "rad.driver.python";
+	private static final String IMAGE_JAVA_SPARK = "rad.driver.java";
+	private static final String IMAGE_SCALA_SPARK = "rad.driver.scala";
+	private static final String IMAGE_OSHINKO_REST = "rad.oshinko.rest";
+	private static final String IMAGE_OSHINKO_CLI = "rad.oshinko.cli";
+	private static final String IMAGE_OSHINKO_WEBUI = "rad.oshinko.webui";
+	private static final String SHOULD_BUILD_ALL_DRIVER_APPLICATION = "rad.build.all.drivers";
+
+	public static final String TEMPLATE_OSHINKO_WEBUI_RESOURCES_URL = "rad.oshinko.web.ui.resources.url";
+	public static final String TEMPLATE_MONGODB_EPHEMERAL_URL = "rad.mongodb.ephemeral.template.json";
+
+	private RadanalyticsConfiguration() {
+		super();
+	}
+
+
+	public static String pySpark() {
```

Comment:
It would be great to add prefix "image" to properties methods, that return image, eg. pySpark() -> imagePySpark().